airflow/providers/cncf/kubernetes/provider.yaml

```diff
@@ -224,6 +224,16 @@ config:
         type: string
         example: ~
         default: "False"
+      worker_pod_pending_fatal_container_state_reasons:
+        description: |
+          If the worker pods are in a pending state due to a fatal container
+          state reasons, then fail the task and delete the worker pod
+          if delete_worker_pods is True and delete_worker_pods_on_failure is True.
+        version_added: ~
```

Comment:
we should add next minor k8s version here

Reply:
version number updated,